locale/de/LC_MESSAGES/django.po

```diff
@@ -115,7 +115,7 @@ msgstr "<b>Das Urheber*innenrecht muss sich ändern, jetzt!</b>"
 
 #: templates/index.html:44
 msgid "How you can take action "
-msgstr "Wie Du aktiv werden kannst"
+msgstr "Wie du aktiv werden kannst"
```

Comment:
This should probably stay the way it was before to maintain consistency with the rest of the translations